<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWidget</class>
 <widget class="QWidget" name="PasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PasswordWidget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>348</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Password Checker</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select operation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="operationList">
          <item>
           <property name="text">
            <string>Validate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hash and Compare</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Validate, Hash, and Compare</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Enter password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Results:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="resultsBox">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Please enter a password above that is greater than the minimum character length.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="changeSourceButton">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Change Comparison Source</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="versionLabel">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
